### ISSUE TITLE
CTC prediction problem in demo.py

### ISSUE THREAD

**Issue by 2113vm** [clovaai/deep-text-recognition-benchmark#185]
I have trained TPS-ResNet-BiLSTM-CTC model on my dataset and have tried to run demo.py. But I had an error in convert.decode():  

```
Traceback (most recent call last):
  File "demo.py", line 133, in <module>
    demo(opt)
  File "demo.py", line 61, in demo
    preds_str = converter.decode(preds_index.data, preds_size.data)
  File "deep-text-recognition-benchmark/utils.py", line 45, in decode
    t = text_index[index, :]
IndexError: too many indices for tensor of dimension 1
```

I solved that when I commented the 60th line of demo.py :

```
preds_index = preds_index.view(-1)
```

And, I think, that the line shouldn't to be, because it's not there for Attn prediction

**Comment by sasi143** [comment]
@2113vm Thanks man....you saved a lot of time of mine

**Comment by gwkrsrch** [comment]
Thank you for raising and resolving this issue:)
This issue has now been resolved by 7f59fb0
